CONTRIBUTING.md

```diff
@@ -0,0 +1,27 @@
+# Contributing
+
+This constitution is a community effort, and everyone is welcome to file issues and submit pull requests.
+
+## How to Contribute
+
+The preferred way to contribute is to submit a Pull Request. A Pull Request is a way to make your modification appear on GitHub in a format that encourages discussion and makes the modification easy to incorporate into the draft document. 
+
+You can find detailed instructions on how to do this [here](SETUP.md).
+
+## What to put in a pull request
+
+Keep your pull request as small as possible. You want the change to be about one topic, usually just one line. If you want to make several different changes, open separate pull requests. It's more work, but this makes it a lot easier for people to review and discuss your changes, and gives you a better chance of having your changes incorporated in the constitution.
+
+If you want to fix a series of typos, you can submit a pull request that incorporates all of your typo changes in the document.
+
+## Adding a line
+
+If you want to add a new, numbered line to the constitution, add it as the last item of the relevant section. This way you won't have to re-number all the other items, which would make it very hard to review the change. Don't worry if the change would make more sense next to a higher-up item. That will be fixed after all the changes are discussed.
+
+## Removing a line
+
+If you'd like to remove a line, delete it but don't update the numbering of any of the other lines. This will be fixed in the end. For now, we want to keep your change restricted to the line you're changing. Make sure you fix any references to the item you deleted.
```

Comment:
I would like to say something like "$II.100 Is obsolete now" instead of just removing the line